pwa-devdocs/scripts/create-reference-docs/templates/renderClassDocs.js

```diff
@@ -0,0 +1,77 @@
+const requiredProp = '<i class="material-icons green">check_box</i>';
+
+/**
+ * Function for overriding template props
+ */
+const overrideProps = (props, propsOverrides) => {
+    if (propsOverrides == undefined) {
+        return props;
+    }
+    let result = Object.assign({}, props);
+
+    let propKeys = Object.keys(props);
+
+    propKeys.forEach(key => {
+        if (propsOverrides[key]) {
+            result[key] = propsOverrides[key];
+        }
+    });
```

Comment:
You could just `const result = Object.assign({}, props, propsOverrides)` to achieve the same result 👍